src/Mod/Sketcher/Gui/TaskSketcherElements.h

```diff
@@ -25,6 +25,7 @@
 #define GUI_TASKVIEW_TaskSketcherElements_H
 
 #include <Gui/TaskView/TaskView.h>
+#include <Mod/sketcher/Gui/RenderingOrderWidget.h>

```

Comment:
```suggestion
#include "RenderingOrderWidget.h"

```

Reply:
https://learn.microsoft.com/en-us/cpp/preprocessor/hash-include-directive-c-cpp?view=msvc-170
the search path is different if you use brackets or quotes

Reply:
Oh damn I missed that. And I spent a lot of time trying to fix that :o

Reply:
Thanks for catching it!